austrakka/components/iam/role/access/funcs.py

```diff
@@ -0,0 +1,62 @@
+from austrakka.components.iam.shared_funcs import _get_default_tenant
+from austrakka.utils.api import api_get, api_post, api_delete
+from austrakka.utils.misc import logger_wraps
+from austrakka.utils.output import print_dict
+
+
+@logger_wraps()
+def get_access(role: str, out_format: str):
+    """
+    Get the list of scope access defined for a role.
+    """
+    tenant_global_id = _get_default_tenant()
+    resp = api_get(path=f"v2/tenant/{tenant_global_id}/role")
+    role_global_id = _get_role_global_id_by_name(resp, role, tenant_global_id)
+
+    resp2 = api_get(
+        path=f"v2/tenant/{tenant_global_id}/role/{role_global_id}/ScopeAccessDefinition",
+    )
+
+    data = resp2['data'] if ('data' in resp2) else resp2
+    print_dict(
+        data,
+        out_format,
+    )
+
+
+@logger_wraps()
+def add_access(role: str, global_ids: list[str]):
+    """
+    Add a new access definition to a role.
+    """
+    tenant_global_id = _get_default_tenant()
+    resp = api_get(path=f"v2/tenant/{tenant_global_id}/role",)
+    role_global_id = _get_role_global_id_by_name(resp, role, tenant_global_id)
+
+    api_post(
+        path=f"v2/tenant/{tenant_global_id}/role/{role_global_id}/ScopeAccessDefinition",
+        data=global_ids,
+    )
+
+
```

Comment:
should remove access have logger wraps? Might be a reason to not have it just checking

Reply:
Will fix.